custom/pnlppgi/reports.py

```diff
@@ -36,6 +36,20 @@ def update_config(config):
     config.update(dict({get_INFilter_element_bindparam('owner_id', idx): val for idx, val in enumerate(users, 0)}))
 
 
+def users_locations():
```

Comment:
i might be missing something but it looks like the way this is used is only to filter the data that was already present in the report and not adding additional locations, no? anyway, i trust that you have thought about and tested this and don't feel the need to block it. I think it would be good to have some comments in the code explaining this a bit better though since someone will have to come back to the PR to try to understand what's going on here.